src/OpenColorIO/OpOptimizers.cpp

```diff
@@ -112,32 +223,30 @@ OCIO_NAMESPACE_ENTER
 
         while (firstindex < static_cast<int>(opVec.size() - 1))
         {
-            ConstOpRcPtr first  = opVec[firstindex];
-            ConstOpRcPtr second = opVec[firstindex + 1];
+            ConstOpRcPtr op1 = opVec[firstindex];
+            ConstOpRcPtr op2 = opVec[firstindex + 1];
+            const auto type1 = op1->data()->getType();
 
-            if (first->canCombineWith(second))
+            if (IsCombineEnabled(type1, oFlags) && op1->canCombineWith(op2))
             {
                 tmpops.clear();
-                first->combineWith(tmpops, second);
+                op1->combineWith(tmpops, op2);
 
                 // tmpops may have any number of ops in it. (0, 1, 2, ...)
-                // (size 0 would occur potentially iff the combination
-                // results in a no-op)
+                // (size 0 would occur potentially iff the combination results in a no-op).
```

Comment:
typo: iff -> if

Reply:
"iff" is a frequent abbreviation (in math and programmer speak) for "if and only if". Is that intentionally what is meant here?  (If it is the intended meaning, then the word "potentially" should be removed, and maybe it's not wise to use that abbreviation.)